portfolio/src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -0,0 +1,33 @@
+package main.java.com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/form-handler")
+public class FormHandlerServlet extends HttpServlet {
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+        // Get the value entered in the form.
+        String textValue = request.getParameter("text-input");
+        String textValue2 = request.getParameter("text-input2");
+        String textValue3 = request.getParameter("text-input3");
+
+        // Print the value so you can see it in the server logs.
+        System.out.println("You submitted: " + textValue);
+        System.out.println("You submitted: " + textValue2);
+        System.out.println("You submitted: " + textValue3);
+
+        // Write the value to the response so the user can see it.
+        response.getWriter().println("You submitted: " + textValue);
+        response.getWriter().println("You submitted: " + textValue2);
+        response.getWriter().println("You submitted: " + textValue3);
+        
+        //Redirects back to the linked webpage
+        response.sendRedirect("http://pabah-sps-summer21.appspot.com/");
```

Comment:
The redirect happens too fast. It just looks like a page refresh, so users will never actually see their input parroted back. 

A neat trick I learned from Sumanjit is that you can add a delay for the redirect using `response.setHeader`. See her code [here](https://github.com/SumanjitG/my-sps-portfolio/blob/main/portfolio/src/main/java/com/google/sps/servlets/ContactMeServlet.java#L34)